python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -318,3 +313,67 @@ def quantize(self, pl_model,  # remove the type requirement for type checking
                     quantized_onnx_model)
         else:
             raise NotImplementedError("Backend {} is not implemented.".format(backend))
+
+    @staticmethod
+    def trace(model: nn.Module, input_sample=None, accelerator=None):
```

Comment:
How about we rename this method to `optimize`? We are also planning to add the new ipex logic here, since in the new ipex version, the usage changes to "model = ipex.optimize(model)".

@jason-dai

Reply:
> How about we rename this method to `optimize`? We are also planning to add the new ipex logic here, since in the new ipex version, the usage changes to "model = ipex.optimize(model)".
> 
> @jason-dai 

We think it better to first merge this pr and maybe change later if necessary.

Reply:
Let's merge the PR first. 

We plan to use `trace` for inference API